app/serializers/base_serializer.rb

```diff
@@ -123,6 +122,18 @@ def show_policy?
     return respond_to_policy && respond_to_manage && can_manage
   end
 
+  def determine_submitter(object)
+    # FIXME: needs to be the creators for assets
+    return object.owner if object.respond_to?('owner')
+    result = object.contributor if object.respond_to?('contributor') && !object.is_a?(Permission)
+    if result
+      return result if result.instance_of?(Person)
+      return result.person if result.instance_of?(User)
```

Comment:
also no longer needed, you can just return the result.  It will always be a Person here